docs/how-to/publishing/android.rst

```diff
@@ -0,0 +1,192 @@
+=======
+Android
+=======
+
+Overview
+--------
+
+The Google Play Store is the most widely-used Android app store. This guide
+focuses on how to distribute a BeeWare app there.
+
+Build the app in release mode
+-----------------------------
+
+Run this command.
+
+.. tabs::
+
+  .. group-tab:: macOS
+
+    .. code-block:: bash
+
+      $ (venv) briefcase package android
+
+  .. group-tab:: Linux
+
+    .. code-block:: bash
+
+      $ (venv) briefcase package android
+
+  .. group-tab:: Windows
+
+    .. code-block:: bash
+
+      C:\...> (venv) briefcase package android
+
+This will result in an Android App Bundle file being generated. An `Android App Bundle
+<https://developer.android.com/guide/app-bundle>`__ is a publishing format that
+includes all your app’s compiled code and resources, and defers APK generation and
+signing to Google.
+
+Sign the Android App Bundle
+---------------------------
+
+.. admonition:: Create code signing identity
+
+  Before you sign the APK files, you need to :doc:`create a code signing
+  identity. <../code-signing/android>`
+
+The Google Play Store requires that the Android App Bundle is signed
+before it is uploaded. Since you already have a code signing identity,
+we run the Java jarsigner tool to sign the AAB file.
+
+In this example below, we assume your code signing identity is stored
+in **upload-key-helloworld.jks** under ``.android`` within your home
+folder.
+
+.. tabs::
+
+  .. group-tab:: macOS
+
+    .. code-block:: bash
+
+      $ ~/.briefcase/tools/java/Contents/Home/bin/jarsigner -verbose -sigalg SHA1withRSA -digestalg SHA1 -keystore ~/.android/upload-key-helloworld.jks android/*/app/build/outputs/bundle/release/app-release.aab upload-key -storepass android
```

Comment:
To confirm - the '*' here is trying to capturie "all .aab files in the android folder" without explicitly naming the app, right? The possible problem here is that a briefcase project can contain multiple apps. 

Also - does this sign-in-place, or produce a new aab file somewhere?

Reply:
> To confirm - the '*' here is trying to capturie "all .aab files in the android folder" without explicitly naming the app, right? The possible problem here is that a briefcase project can contain multiple apps.

True -- I'm trying to capture "all .aab files in the android folder". And true -- this is a bad fit given the possibility of a briefcase project containing multiple apps. I hadn't considered that.

I'll fix that in a follow-up push in a sec. 

> Also - does this sign-in-place, or produce a new aab file somewhere?

In-place.

Reply:
96366039dd85175f7cb2ac6e352a8f4381049c4d